irods_catalog_provider/Dockerfile

```diff
@@ -38,6 +38,7 @@ RUN apt-get update && \
         irods-database-plugin-postgres=${irods_package_version} \
         irods-runtime=${irods_package_version} \
         irods-server=${irods_package_version} \
+        irods-resource-plugin-s3 \
```

Comment:
I see that the S3 resource plugin is installed, but not configured. Is that on purpose?

The `=<package_version>` is missing as well. We should lock that down.

Reply:
Yeah, there no config in iRODS because there's no procreated bucket in MinIO. I thought it's the similar approach to iRODS Web UI accounts - there's a MinIO admin account which can be used by iRODS S3 resource plugin but if you have multiple resources then everyone will have complete control over all MinIO buckets. We could add a bucket and non-admin account.

I copied the same tag which didn't work but now I realize - I should have checked plug-in-specific version. Another tag is necessary, I suppose?

Reply:
maybe we don't need to pin it - as the current edit will pull/install the latest s3 plugin that is compatible with the iRODS server version?   Hmm... that may not be true...

Reply:
It appears the same tag *should* work, but I did try that initially and got an error when building - that's why I removed it.

```
root@3cc28af82a1e:/# sudo apt-cache policy irods-resource-plugin-s3
irods-resource-plugin-s3:
  Installed: 4.3.0.0-1~focal
  Candidate: 4.3.0.0-1~focal
  Version table:
 *** 4.3.0.0-1~focal 500
        500 https://packages.irods.org/apt focal/main amd64 Packages
        100 /var/lib/dpkg/status

root@3cc28af82a1e:/# sudo apt-cache policy irods-server
irods-server:
  Installed: 4.3.0-1~focal
  Candidate: 4.3.0-1~focal
  Version table:
 *** 4.3.0-1~focal 500
        500 https://packages.irods.org/apt focal/main amd64 Packages
        100 /var/lib/dpkg/status
```

Reply:
Subtle - note the extra `0` on the plugin version.    Our plugins are tied to the server version, but have an additional fourth value so they can be versioned independently from the server.

Reply:
LOL, I noticed that as well. 

I added another `ARG` but it's not very pretty.

Can you change/edit my PR the way you want it to be? I know it's possible (PR edits by maintainer) but I don't know how to do it myself.

Reply:
yes, we can push to your branch -- we'll test this out today/tomorrow and make it real nice.

Reply:
i will note that this PR is coming from your main branch - so we will be double-careful before we force-push anything.

Reply:
I'll just leave my fork as-is, but it's perfectly fine if you make own commit and then I can close this PR - I don't mind if that's more convenient.